roles/kubernetes/preinstall/tasks/etchosts.yml

```diff
@@ -9,6 +9,7 @@
     create: yes
     backup: yes
     marker: "# Ansible inventory hosts {mark}"
+  ignore_errors: yes
```

Comment:
Could we use `failed_when: false` instead? Just to not spoil build logs with reds :)